app/lib/actions/middleware/backend_services_check.rb

```diff
@@ -24,6 +25,11 @@ def plan(*args)
 
       protected
 
+      def capsule_id(args)
+        capsule_hash = args.select { |x| x[:capsule_id] }
```

Comment:
I'm not sure you can guarantee that all args are a hash.  You may want to check that 'x' is a hash first.